docs/CreateEdit.md

```diff
@@ -321,7 +321,7 @@ const ageValidation = (value, values) => {
     if (!value) {
         return 'The age is required';
     }
-    if (age < 18) {
+    if (values.age < 18) {
```

Comment:
Agreed, this example is wrong. About your question, the parameters are `(value, allValues)`. So here, `age` should be `value`.